index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="assets/stylesheets/main.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+  <p class="welcome">Welcome</p>
+  <nav class="mobile-menu"><i class="fa fa-bars"></i></nav>
+  <header class="header-mobile">
+    <h1 class="black-text-header">Hey there. I’m<br> Jorge<br><span>I’m a software<br>
+        developer</span>
+    </h1>
+    <p class="header-paragraph"> I can help you build a product ,<br> feature or website Look through<br> some of my
+      work and
+      experience!<br> If you like what you see and have<br> a project you need coded, <br>don’t hesitate to contact me.
+    </p>
+    <div class="social-media-icons icons-top">
+      <a href="#"><img src="/assets/images/twitter.svg"></a>
+      <a href="#"><img src="/assets/images/linkedin.svg"></a>
+      <a href="#"><img src="/assets/images/medium.svg"></a>
+      <a href="#"><img src="/assets/images/github.svg"></a>
+      <a href="#"><img src="/assets/images/angellist.svg"></a>
```

Comment:
- [ ] In order to display your assets in other browsers you need to use relative paths in your src attributes. Kindly add these:
```suggestion
      <a href="#"><img src="./assets/images/twitter.svg"></a>
      <a href="#"><img src="./assets/images/linkedin.svg"></a>
      <a href="#"><img src="./assets/images/medium.svg"></a>
      <a href="#"><img src="./assets/images/github.svg"></a>
      <a href="#"><img src="./assets/images/angellist.svg"></a>
```